### DISCUSSION TITLE
Setting up file shares (pull from camera)

### DISCUSSION THREAD

**Discussion by clee604** [gtxaspec/wz_mini_hacks#135] [Q&A] [answered]
Maybe this is a dumb question but I haven't been to figure it out through googling.

Here are the main thing I want to do:
1. Be able to pull recorded videos from the camera's SD card via network using a Windows PC, ideally with a user/password authentication.

I'm assuming CIFS = SAMBA so I should be enabling that (NFSv4 is for Linux? Or something else?).

After that, I'm not sure what else I am supposed to configure to set this up. I've set up a SAMBA share on a linux host before, but that is using the smbd.service which doesn't seem to apply here.

**Comment by dudududodododedede** [comment]
you can also accomplish this without a share and just connect to the camera through WinSCP

**Reply by clee604** [reply]
I actually tried this first, but I get this error
![image](https://user-images.githubusercontent.com/93049965/175834186-836368a1-16ae-4bad-832e-154c7716ae14.png)

**Comment by clee604** [comment, marked as answer]
For those trying to get WinSCP to work, after much trial and error, the issue is this setting. 

Set to /bin/sh, I believe it defaults to bash which errors out

![image](https://user-images.githubusercontent.com/93049965/176284822-98cf5dce-5eb8-438b-bd65-8126294d47c7.png)

**Comment by clee604** [comment]
I noticed that SCP caps out at around 1.6mb/s for file transfers. Are there ways to improve speed? Would CIFS be faster?
